src/components/human-impact-layers/human-impact-layers-component.jsx

```diff
@@ -1,39 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 // Constants
-import { VIEW_MODE } from  'constants/google-analytics-constants';
-import { layersConfig } from 'constants/mol-layers-configs';
-import { LAND_HUMAN_PRESSURES_IMAGE_LAYER } from 'constants/layers-slugs';
+import { layersConfig, LAYERS_CATEGORIES } from 'constants/mol-layers-configs';
 import { humanPressuresLandUse } from 'constants/human-pressures';
 // Utils
-import { handleLayerRendered, layerManagerVisibility, handleLayerCreation } from 'utils/layer-manager-utils';
-import { humanPressuresPreloadFixes, dispatchLandPressuresLayersAnalyticsEvents } from 'utils/raster-layers-utils';
```

Comment:
We don't use functions like `handleLayerRendered`, `toggleWDPALayer` or  `getToggledLayer` from  'utils/layer-manager-utils.js' anywhere, do you think we can remove them safely?